server/controller/AppsController.js

```diff
@@ -1,20 +1,57 @@
-import mongoose from 'mongoose';
 import responseLib from '../lib/responseLib';
 import * as actionStatus from '../constants/actionStatus';
+import IntegrationConfig from '../thirdparty/integrationConfig';
+import AuthorizedApps from '../models/AuthorizedApps';
 
-const AppsCollection = mongoose.model('Apps');
+const getApps = (req, res) => {
+  const appsAndEvents = [];
+  for (const [appName, appData] of Object.entries(IntegrationConfig)) {
+    const singleAppDetails = { Trigger: [], Action: [] };
+    singleAppDetails.appName = appName;
+    singleAppDetails.icon = appData.icon_path;
+    for (const event of appData.Events) {
+      singleAppDetails[event.EventType].push(event.EventName);
+    }
+    appsAndEvents.push(singleAppDetails);
+  }
+// const AppsCollection = mongoose.model('Apps');
 
-const createApp = async (req, res) => {
-  console.log(req.body);
-  const createdApp = await AppsCollection.create(req.body);
-  res.send(createdApp);
+  const generatedResponse = responseLib.generateResponse(false, actionStatus.SUCCESS,
```

Comment:
pass in an object with values instead of multiple params. It's easier to update/ changes params later.
Also the code is more readable